### ISSUE TITLE
Support query without FROM clause

### ISSUE THREAD

**Issue by dai-chen** [opendistro-for-elasticsearch/sql#264]
This is helpful for documentation improvement work in https://github.com/opendistro-for-elasticsearch/sql/issues/243. In most database documentation, sample use of function is as simple as `SELECT function(literal)`. In this case, the result set is just 1 row which is very clear to show what's the result of function call.

However, it is not supported in our plugin because the current code path always assumes and requires an index name in FROM to execute the DSL against.

Example in MySQL doc: https://dev.mysql.com/doc/refman/8.0/en/mathematical-functions.html#function_log

```
mysql> SELECT LOG(2,65536);
        -> 16
mysql> SELECT LOG(10,100);
        -> 2
mysql> SELECT LOG(1,100);
        -> NULL
```

**Comment by dai-chen** [comment]
https://github.com/opendistro-for-elasticsearch/sql/issues/64#issuecomment-572319709

**Comment by dai-chen** [comment]
The support is already added into new query engine with arithmetic expression and one function `abs` for now. Docs for introducing the new query engine will be added soon. We're adding more SQL functions at the moment so resolving this issue for now..

**Comment by mararual** [comment]
Hello. Was this fixed released already? We are on ES 7.9 and Open Distro 1.11.0 but the issue persists.

**Comment by dai-chen** [comment]
@mararual The issue has been fixed in our new query engine. It was experimental and just enabled in `develop` branch. For ODFE 1.11.0, you can enable it manually by changing the plugin setting to `true`: https://github.com/opendistro-for-elasticsearch/sql/blob/master/docs/user/admin/settings.rst#opendistro-sql-engine-new-enabled. Thanks!
